Connector Documentation - Adyen and Banked (#27)
gr4vy/gr4vy-docs: docs/connectors/adyen-card.mdx

---
title: Adyen
sidebar_label: Adyen
slug: /connectors/adyen-card
---
import Image from "@site/src/modules/Image";
import api_key_image from "./assets/adyen_api_key.png";
import merchant_account_image from "./assets/adyen_merchant_account.png";

## Sign-up
Adyen provides a self-service sign-up for a sandbox account.  To sign-up for an account simply visit the [sign-up page](https://www.adyen.com/signup) and fill in the details.

## Credentials
When setting up Adyen in the Gr4vy Dashboard, you will need to configure the following credentials from Adyen:

> * **API Key**

Reviewer: ```suggestion
### API Key
```

This way we can link to this section.

  The Adyen API key is obtained from the Adyen Admin Portal under the `Developers` -> `API Credentials` heading.

  <Image src={api_key_image} title="Adyen API Key" />

> * **Merchant Account**

Reviewer: ```suggestion
### Merchant Account
```

  The Adyen Merchant Account can be found in the Adyen Admin Portal under the `Account` -> `Merchant accounts` heading.

  <Image src={merchant_account_image} title="Adyen API Key" />

> * **Live endpoint prefix (conditional)**

Reviewer: ```suggestion
### Live endpoint prefix (conditional)
```

  
  The live endpoint prefix is required for live Adyen transactions.  The prefix is a combination of the random and company name from the live endpoint.

  For example, if your live URL is `https://1797a841fbb37ca7-AdyenDemo-checkout-live.adyenpayments.com/checkout/v53/payments` then the live URL prefix would be `1797a841fbb37ca7-AdyenDemo`.

See [Live URL prefix](https://docs.adyen.com/development-resources/live-endpoints#live-url-prefix) for more details.

> * **Mode**

Reviewer: ```suggestion
### Mode
```

---
title: Adyen
sidebar_label: Adyen
slug: /connectors/adyen-card
---
import Image from "@site/src/modules/Image";
import api_key_image from "./assets/adyen_api_key.png";
import merchant_account_image from "./assets/adyen_merchant_account.png";

## Sign-up
Adyen provides a self-service sign-up for a sandbox account.  To sign-up for an account simply visit the [sign-up page](https://www.adyen.com/signup) and fill in the details.

## Credentials
When setting up Adyen in the Gr4vy Dashboard, you will need to configure the following credentials from Adyen:

* **API Key**

  The Adyen API key is obtained from the Adyen Admin Portal under the `Developers` -> `API Credentials` heading.

  <Image src={api_key_image} title="Adyen API Key" />

* **Merchant Account**

  The Adyen Merchant Account can be found in the Adyen Admin Portal under the `Account` -> `Merchant accounts` heading.

  <Image src={merchant_account_image} title="Adyen API Key" />

* **Live endpoint prefix (conditional)**
  
  The live endpoint prefix is required for live Adyen transactions.  The prefix is a combination of the random and company name from the live endpoint.

>   For example, if this was your live URL:
>   
>   * https://1797a841fbb37ca7-AdyenDemo-checkout-live.adyenpayments.com/checkout/v53/payments
> 
>   Then the live URL prefix would be
>   
>   * `1797a841fbb37ca7-AdyenDemo`
> 
>   See [Live URL prefix](https://docs.adyen.com/development-resources/live-endpoints#live-url-prefix) for more details.

Reviewer: ```suggestion
  For example, if your live URL is `https://1797a841fbb37ca7-AdyenDemo-checkout-live.adyenpayments.com/checkout/v53/payments` then the live URL prefix would be `1797a841fbb37ca7-AdyenDemo`.

See [Live URL prefix](https://docs.adyen.com/development-resources/live-endpoints#live-url-prefix) for more details.
```

Generally how I'd write this.

* **Mode**
  

>   The mode is used to configure either `Live` or `Sandbox` credentials.

Reviewer: ```suggestion
  The mode is used to configure if the credentials are for usage with the `Live` or `Sandbox` APIs.
```
